src/Compiler/CompilerOpenFPGA.cpp

```diff
@@ -492,6 +492,25 @@ bool CompilerOpenFPGA::RegisterCommands(TclInterpreter* interp,
   };
   interp->registerCmd("flat_routing", flat_routing, this, 0);
 
+  auto use_vpr_latest = [](void* clientData, Tcl_Interp* interp, int argc,
+                           const char* argv[]) -> int {
+    CompilerOpenFPGA& comp = *(CompilerOpenFPGA*)clientData;
+    std::filesystem::path vpr = comp.m_vprExecutablePath.lexically_normal();
+    constexpr bool dbg = false;
+    if (dbg) std::cerr << "\n  VPR WAS: " << vpr << '\n';
```

Comment:
Please use this instead.
https://github.com/os-fpga/FOEDAG/blob/main/src/Compiler/CompilerOpenFPGA.cpp#L2906C7-L2906C19